source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -156,20 +156,36 @@ def sort_journeys(resp, journey_order, clockwise):
         resp.journeys.sort(journey_sorter[journey_order](clockwise=clockwise))
 
 
-def tag_journeys(resp):
+def tag_journeys(resp, request, instance):
     """
     qualify the journeys
     """
     car = next((j for j in resp.journeys if helpers.is_car_direct_path(j)), None)
+    co2_emission_value = 0
+    co2_emission_unit = 0
     if car is None or not car.HasField('co2_emission'):
-        return
+        # if there is no car journey found, we request kraken to give us an estimation of
+        # co2 emission
+        from jormungandr.georef import Kraken
+        co2_estimation = Kraken(instance).get_car_co2_emission_on_crow_fly(request['origin'], request['destination'])
```

Comment:
The instance should already have one object georef, if not you need to add it. there Is no point to abstract kraken if we instantiate the class every time we need it.